.github/workflows/simulation.yml

```diff
@@ -0,0 +1,29 @@
+name: simulate quicksilver
+
+on:
+  pull_request:
+  push:
+    paths-ignore:
+      - '*.md'
+    branches:
+      - main
```

Comment:
simulation in general take a lot of ci minutes do we want to run it on every push? can this be a trigger that we can run once pr is ready to be merged with all sanity checks?

Reply:
I'm not sure how, but I've seen actions trigger on application of a 'ready-to-merge' type label. Is this easy to do?

Reply:
i have also seen label based let me dig in a little